### PR TITLE
Fix failing tests

### PR DESCRIPTION
**Pull request by sam09**
- [x] Improvement
    - [x] Bug fix: **`DurationCalculator: Fix 3327`**
    
Instant Answer Page: https://duck.co/ia/view/duration_calculator

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: t/DurationCalculator.t [hunk 1]
```diff
@@ -46,11 +47,11 @@ my $invalid_query1 = "2 day 2 day + 4 day";
 my $invalid_query2 = "1 day + 1 hay";
 my $invalid_query3 = "-54minutes - 39";
 my $invalid_query4 = "4 hours";
-
+Test::MockTime::set_fixed_time("06/02/2016 02:00:00", "%m/%d/%Y %H:%M:%S");
```
Comment (GuiltyDolphin) [inline t/DurationCalculator.t]: Lets just use `2016-06-02T02:00:00Z`.

**Comment by GuiltyDolphin** [comment]
@sam09 Thanks! :dolphin:

**Comment by moollaza** [comment]
@sam09 thanks for the super quick fix! 💯

**Pull request merged**